<launch>

  <node pkg="nav_to_goal" type="nav_to_goal" name="nav_to_goal" output="screen">
      <param name="max_angular_velocity" value="0.07" />
      <param name="distance_min" value="0.5" />
      <param name="k_vel" value="5.0" />
      <param name="vel_max" value="0.3" />
      <param name="k_alpha" value="0.1" />
      <param name="TARGET_TF" value="base_link" />
      <param name="SOURCE_TF" value="odom" />
      <param name="goal_x" value="-1.0" />
      <param name="goal_y" value="-2.0" />

      <remap from="/nav_to_goal/nao_vel" to="/rovio/cmd_vel"/>
  </node>
</launch>
